libcxx/test/libcxx/depr/exception.unexpected/unexpected_disabled_cpp17.verify.cpp

```diff
@@ -8,6 +8,11 @@
 
 // UNSUPPORTED: c++03, c++11, c++14
 
+// When built with modules, this test gives diagnostics like declaration of
+// 'unexpected' must be imported from module 'std.expected.unexpected' before
+// it is required. Therefore disable it in this configuration.
+// UNSUPPORTED: clang-modules-build
```

Comment:
Doesn't this problem also arise with every other `.verify.cpp` test we have that checks for the absence of a type or function?